README.md

```diff
@@ -1,2 +1,24 @@
-# hw01
-Repository for homework 01
+# Hi, I am **Jiaqian(Jane) Wang**:)
+
+![](汪嘉倩-head.jpg)
+
+### **Personal Facts**
+* **Birthday**: I was born on 01/19.
+* **Hometown**: My hometown is Shanghai, China. But I was born and grew up in [Tianjin](https://en.wikipedia.org/wiki/Tianjin), a coastal metropolis in northern China.
+* **Hobbies**: In my spare time, I enjoy reading, painting and watching films.
+* **My Favorite Quote**:  [*There is a crack in everything, that's how the light gets in.*](https://qz.com/835076/leonard-cohens-anthem-the-story-of-the-line-there-is-a-crack-in-everything-thats-how-the-light-gets-in/)
+
+### **Research Interest**    
+Broadly, my research focuses on following domains in marketing and policy-making context.
+* Judgment and decision-making
+* Information processing    
+
+### **Education Background**
+* 2018-2019(Expected)    
+  **M.A. in Social Sciences (Psychology)**    
+  The University of Chicago, USA
+* 2014-2018           
+  **B.M in Marketing** & **B.A. in History and Theories of Art**     
+  Peking University, China    
+
+### **[Contact Me](jiaqianwang@uchicago.edu)**
```

Comment:
I get an error page when I try to click on "Contact Me." I think it may not work with an e-mail address link, only web page urls. Alternatively, you could just write your e-mail out on the bottom.